src/ai/lua/core.cpp

```diff
@@ -498,6 +498,14 @@ static int cfun_ai_get_recruitment_pattern(lua_State *L)
 	return 1;
 }
 
+static int cfun_ai_get_retreat_enemy_weight(lua_State *L)
```

Comment:
Same here, kinda silly to add something already deprecated.

Reply:
Also done in 200f5dd913